fv3core/pace/fv3core/stencils/dyn_core.py

```diff
@@ -691,7 +691,7 @@ def _checkpoint_dsw_in(self, state: DycoreState):
                 ucd=state.uc,
                 vcd=state.vc,
                 wd=state.w,
-                delpcd=self.delpc,
+                delpcd=self._vt,
```

Comment:
It is wild to me that delpc is still a temporary but it's not the variable in the savepoint.

Reply:
Should we add some # commenting about this? It's very confusing.